config/router.js

```diff
@@ -11,6 +11,7 @@ module.exports = function (router) {
     router.get('/channels', c.channel.get_channels);
     //router.get('/channels/:id', c.channel.get_channel);
     router.get('/channels/:id/growth', c.growth.get_channel);
+    router.get('/channels/growth', c.growth.get_grouped_channel);
```

Comment:
add space to methods that use three characters.
```javascript
router.get (...)
router.put (...)
router.del (...)
router.post(...)
```

so we can maintain the alignment.